pytest.ini

```diff
@@ -2,6 +2,6 @@
 addopts = --doctest-modules
 doctest_optionflags = ELLIPSIS
 testpaths = sasmodels
-python_files = *.py
+python_files = doc/**/*.py example/**/*.py sasmodels/**/*.py
```

Comment:
I think is a good idea - seems appropriate to test the python files in *this* library, not every library that this library uses.